Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst

```diff
@@ -39,6 +39,22 @@ hierarchy of these two arrays.
 .. index:: DataHandler; Commands array
 .. _tce-commands:
 
+Basic Usage
+===========
+
+.. highlight:: php
+
+::
+
+   use TYPO3\CMS\Core\Utility\GeneralUtility;
+   use TYPO3\CMS\Core\DataHandling\DataHandler;
+   
+   $dataHandler = GeneralUtility::makeInstance(DataHandler:class);
+   
+   $cmd = [];
+   $data = [];
+   $dataHandler->start($data, $cmd);
```

Comment:
Maybe we should also add a few words about this above this example.

Reply:
Feel free to do so in another PR?